src/site/stages/build/plugins/parse-html.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-param-reassign */
+const path = require('path');
+const cheerio = require('cheerio');
+
+const parseHtml = files => {
+  for (const fileName of Object.keys(files)) {
+    if (path.extname(fileName) === '.html') {
+      files[fileName].parsedContent = cheerio.load(files[fileName].contents);
```

Comment:
I like `parsedContent` but `file.$` or `file.dom` could be kinda snazzy

Reply:
+1 for file.dom

Reply:
`file.dom` it is! I want it to be obvious by just reading it, and `file.$` doesn't give much away. I like `file.dom`. :+1: